README.md

```diff
@@ -52,7 +50,7 @@ You can start using go-libp2p in your Go application simply by adding imports fr
 import "github.com/libp2p/go-libp2p"
 ```
 
-### Examples
+## Examples
```

Comment:
According to the table of contents, this is still a subsection of Usage.

Reply:
Usage also changed from heading level 2 to heading level 1